src/app/@theme/components/social-links-component/social-links-component.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'intelowl-social-links-component',
```

Comment:
can you just remove the `-component` from the name everywhere ? didn't notice it before but is pretty redundant.

Reply:
My bad, it does look weird